xfce-extra/xfce4-whiskermenu-plugin/xfce4-whiskermenu-plugin-1.6.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="Alternate application launcher for Xfce"
+HOMEPAGE="https://gottcode.org/xfce4-whiskermenu-plugin"
+SRC_URI="https://github.com/gottcode/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="
+	virtual/libintl:=
+	x11-libs/gtk+:2=
+	xfce-base/exo:=
+	xfce-base/garcon:=
+	xfce-base/libxfce4ui:=
+	xfce-base/libxfce4util:=
+	xfce-base/xfce4-panel:="
+
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
```

Comment:
also I just noticed - if you use **cmake-utils**, you should use `cmake-utils_src_prepare` instead of `default`. Sorry, this was an oversight on my side.